charts/hivemq-platform/values.yaml

```diff
@@ -182,8 +182,16 @@ services:
     keystoreSecretKey: ""
     # The inlined password for the keystore, Base 64 encoded.
     keystorePassword: ""
-    # Alternatively, the name of the secret with the password, using the key: `keystore.password`.
+    # The inlined private key password for the keystore, Base 64 encoded. Fallbacks to the `keystorePassword` value if empty or not set.
+    keystorePrivatePassword: ""
+    # Alternatively, the name of the secret to use to fetch:
+    # - The keystore password by using the key defined in the `keystorePasswordSecretKey` value. Defaults to `keystore.password`
```

Comment:
This is confusing, the description should probably move a couple of lines down to where the variable is.

Reply:
Yeah, that's true. I made it simpler now, but I think it's a little clearer now - I hope. If not, let me know and maybe I can include an example here or even in the official doc?

Reply:
I think it would be good to have some complete examples in the docs (with and w/o secret, with and w/o private key password etc.). We have those scenarios "hidden" in the tests, but I think most people are not used to Helm unit tests and how to read them. So I think the official documentation is a better place.